File: week02/day_2/draw-triangle.py
__author__ = 'ylwoi'
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

x = int(input("Enter a number to to set triangle size:"))
for i in range(1, x+1):
    print("*"*i)

File: week07/day2/draw-triangle.js
/**
 * Created by cw on 2017-05-02.
 */
'use strict';

var lineCount = 4;
var triangle = '*';

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is
for (var i = 1; i <= lineCount;i++) {
    console.log(triangle.repeat(i));
}